.github/workflows/solutions-dpg-qa.yaml

```
@@ -0,0 +1,156 @@
+name: Pucar-Install Workflow Solutins-dpg-qa
+on:
```

yamllint

[warning] truthy value should be one of [false, true]

(truthy)